src/index.js

```diff
@@ -581,6 +580,7 @@ class KadDHT extends EventEmitter {
    * @param {PeerId} id
    * @param {Object} options - findPeer options
    * @param {number} options.timeout - how long the query should maximally run, in milliseconds (default: 60000)
+   * @param {AbortControllerSignal} options.signal - an `AbortController` signal
```

Comment:
Does the DHT constructor need an AbortControllerSignal parameter?

Reply:
I think I will remove this from the findPeer method until it actually supports it.

> I'm not sure I'd include the AbortController logic until we move to an async/await API

Do you think we should just remove it from randomWalk altogether, or leave what's in there? When we do the update in async/await if `findPeer` takes `options.signal` as a parameter, we won't have to do any work there to hook up randomWalk. Adding an abort test to findPeer will verify it's working, since randomWalk is validating it's passing the param onward.